memor/params.py

```diff
@@ -46,3 +47,57 @@ class RenderFormat(Enum):
     DICTIONARY = "DICTIONARY"
     ITEMS = "ITEMS"
     DEFAULT = STRING
+
+
+class LLM_MODEL(Enum):
```

Comment:
`LLM_MODEL` is typically used for constants, not classes or enums, in Python. Since you're defining an Enum the conventional style is **CamelCase**. My suggestion: `LLMModel`.